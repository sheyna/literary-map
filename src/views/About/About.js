import { useEffect } from 'react';
import PropTypes from 'prop-types';

import styles from './About.module.css';

function About({setShowDecorativeTitle}) {

  useEffect(function() {
    setShowDecorativeTitle(false);
  }, [setShowDecorativeTitle]);

  return (
    <section className={styles.About}>
      <h1>About</h1>
      <p>Take a virtual tour of some of England's literary sites.</p>

      <p>Probject by Sheyna Watkins</p>
    </section>
  )
}

About.propTypes = {
	setShowDecorativeTitle: PropTypes.func
};

export default About
