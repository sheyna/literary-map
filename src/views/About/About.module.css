.About h1 {
  font-size: 1.5rem;
  font-weight: 600;
}
