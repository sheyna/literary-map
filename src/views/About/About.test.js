import ReactTestRenderer from 'react-test-renderer';
import About from  './About';

describe('SiteFooter component', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(
      <About />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
