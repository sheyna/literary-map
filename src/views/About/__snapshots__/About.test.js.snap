// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteFooter component should match snapshot 1`] = `
<section
  className="About"
>
  <h1>
    About
  </h1>
  <p>
    Take a virtual tour of some of England's literary sites.
  </p>
  <p>
    Probject by Sheyna Watkins
  </p>
</section>
`;
